### PR TITLE
PM feedbacks

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?
All feedbacks from log PM.

### Motivation

To perfectly match PM expectations and documentation

### Preview link

https://docs-staging.datadoghq.com/gus/log-update/logs/

### CODE REVIEW

File: content/logs/explore.md [hunk 1]
```diff
@@ -84,14 +84,6 @@ Once this is done, the value of this attribute is stored **for all new logs** an
 Use facets to easily filters on your logs. The search bar and url automatically reflect your selections.
 {{< img src="logs/explore/facet_panel.png" alt="Facet panel" responsive="true" popup="true">}}
 
-### Expanded view
```
Comment (NBParis) [inline content/logs/explore.md]: We should replace this by a section on the multiline display

File: content/logs/languages/ruby.md [hunk 2]
```diff
@@ -120,17 +120,16 @@ Create a `ruby.d/conf.yaml` file in your `conf.d/` folder with the following con
...
-    #   tags: (optional) add tags to each logs collected
+    ## - type : file (mandatory) type of log input source (tcp / udp / file)
+    ##   port / path : (mandatory) Set port if type is tcp or udp. Set path if type is file
+    ##   service : (mandatory) name of the service owning the log
+    ##   source : (mandatory) attribute that defines which integration is sending the logs
+    ##   sourcecategory : (optional) Multiple value attribute. Can be used to refine the source attribtue
+    ##   tags: (optional) add tags to each logs collected
 
   - type: file
     path: /path/to/your/ruby/log.log
```
Comment (NBParis) [inline content/logs/languages/ruby.md]: @l0k0ms There are reference to logmatic in this article.

File: content/logs/explore.md (hunk 1 above)
Comment (l0k0ms) [reply content/logs/explore.md]: 👌

File: content/logs/languages/ruby.md (hunk 2 above)
Comment (l0k0ms) [reply content/logs/languages/ruby.md]: 👌

**Pull request merged**